### PR TITLE
UIBULKED-66: Progress bar for uploading identifier files and retrieving relevant data

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
Progress bar for uploading identifier files and retrieving relevant data

## Purpose

https://issues.folio.org/browse/UIBULKED-66
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js [hunk 1]
```diff
@@ -92,6 +95,10 @@ export const BulkEditInApp = ({ title, onContentUpdatesChanged }) => {
     }]);
   };
 
+  const getIsTemporaryLocation = ({ option }) => option === OPTIONS.TEMPORARY_LOCATION;
+
+
+
```
Comment (uladislausamets) [inline src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js:101]: 👀

File: src/components/BulkEditActionMenu/BulkEditActionMenu.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ const BulkEditActionMenu = ({
   const renderStartBulkEditButtons = () => {
     return (
       <>
-        {isStartBulkInAppActive && (
+        {isStartBulkInAppActive && !isCompleted && (
```
Comment (uladislausamets) [inline src/components/BulkEditActionMenu/BulkEditActionMenu.js:103]: you can add it [here](https://github.com/folio-org/ui-bulk-edit/pull/103/files#diff-ba4586c0afa30a5c1d713f14228b91194260a28b1da945dc7ca296567b468598R98)

**Review by vashjs — APPROVED** [review]

File: src/components/BulkEditActionMenu/BulkEditActionMenu.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/components/BulkEditActionMenu/BulkEditActionMenu.js]: Thanks a lot! Already fixed it.

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js]: Ye, my bad. Removed this empty string already.

**Review by uladislausamets — APPROVED** [review]

**Pull request merged**